src/client/driver/command-executors/execute-action-command.js

```diff
@@ -148,6 +152,15 @@ function ensureCommandElements (command, timeout) {
         });
 }
 
+function ensureCommandArguments (command) {
+    if (command.type === COMMAND_TYPE.pressKey) {
+        var parsedKeySequence = parseKeySequence(command.keys);
```

Comment:
Actually, we can do it on server-side.

Here is `parseKeySequence`:
```js
import hammerhead from '../../../deps/hammerhead';
import testCafeCore from '../../../deps/testcafe-core';
import { getKeyArray, getSanitizedKey } from './utils';
import KEY_MAPS from '../../../utils/key-maps';

var trim       = hammerhead.utils.trim;
var arrayUtils = testCafeCore.arrayUtils;

export default function (keyString) {
    keyString = trim(keyString).replace(/\s+/g, ' ');

    var keyStringLength = keyString.length;
    var lastChar        = keyString.charAt(keyStringLength - 1);
    var charBeforeLast  = keyString.charAt(keyStringLength - 2);

    // NOTE: trim last connecting '+'
    if (keyStringLength > 1 && lastChar === '+' && !/[\+ ]/.test(charBeforeLast))
        keyString = keyString.substring(0, keyString.length - 1);

    var combinations = keyString.split(' ');

    var error = arrayUtils.some(combinations, combination => {
        var keyArray = getKeyArray(combination);

        return arrayUtils.some(keyArray, key => {
            var isChar          = key.length === 1 || key === 'space';
            var sanitizedKey    = getSanitizedKey(key);
            var modifierKeyCode = KEY_MAPS.modifiers[sanitizedKey];
            var specialKeyCode  = KEY_MAPS.specialKeys[sanitizedKey];

            return !(isChar || modifierKeyCode || specialKeyCode);
        });
    });

    return {
        combinations: combinations,
        error:        error,
        keys:         keyString
    };
}
```

Below is `getSanitizedKey` and `getKeyArray`:
```js
export function getKeyArray (keyCombination) {
    // NOTE: we should separate the '+' symbol that concats other
    // keys and the '+'  key to support commands like the 'ctrl++'
    var keys = keyCombination.replace(/^\+/g, 'plus').replace(/\+\+/g, '+plus').split('+');

    return arrayUtils.map(keys, key => key.replace('plus', '+'));
}

export function getSanitizedKey (key) {
    var isChar       = key.length === 1 || key === 'space';
    var sanitizedKey = isChar ? key : key.toLowerCase();

    if (KEY_MAPS.modifiersMap[sanitizedKey])
        sanitizedKey = KEY_MAPS.modifiersMap[sanitizedKey];

    return sanitizedKey;
}
```

And finally `KEY_MAPS.modifiersMap`, `KEY_MAPS.modifiers` and `KEY_MAPS.specialKeys:
```js
const MODIFIERS = {
    alt:   18,
    ctrl:  17,
    meta:  91,
    shift: 16
};
...
const SPECIAL_KEYS = {
    backspace: 8,
    capslock:  20,
    delete:    46,
    down:      40,
    end:       35,
    enter:     13,
    esc:       27,
    home:      36,
    ins:       45,
    left:      37,
    pagedown:  34,
    pageup:    33,
    right:     39,
    space:     32,
    tab:       9,
    up:        38
};
...
modifiersMap: {
     option: 'alt'
},
```
It's more like a proposal.
What do @inikulin and @AlexanderMoskovkin think about it?